public/js/scripts.js

```diff
@@ -0,0 +1,29 @@
+/*!
+* Start Bootstrap - Clean Blog v6.0.7 (https://startbootstrap.com/theme/clean-blog)
+* Copyright 2013-2021 Start Bootstrap
+* Licensed under MIT (https://github.com/StartBootstrap/startbootstrap-clean-blog/blob/master/LICENSE)
+*/
+window.addEventListener('DOMContentLoaded', () => {
+    let scrollPos = 0;
+    const mainNav = document.getElementById('mainNav');
+    const headerHeight = mainNav.clientHeight;
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/AuroreFunes/OCP5_blog/pullRequest?prid=9269757)
```suggestion
  const headerHeight = mainNav.clientHeight;
```